docroot/themes/custom/uids_base/templates/navigation/superfish.html.twig

```diff
@@ -16,9 +16,9 @@
  */
 #}
 
-<nav role="navigation" aria-label="Site main navigation">
+<nav aria-label="Site main">
```

Comment:
Sorry to nitpick this and maybe you're reading indicated this was the right way to go. I think the removal of the word navigation from the aria-label seems incorrect. This take stems from reading through the W3C disclosure menu example: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/

In that example, they just put the name of the organization/website in the aria-label, which is an odd example I'll admit, but that makes me think that the label isn't necessarily interpreted as navigation-related. Since we have multiple menus on many pages, it seems appropriate to specify that it is navigation in the label.

If this change is based on some other specific guidance, feel free to ignore this.

Reply:
```suggestion
<nav aria-label="Main">
```